### DISCUSSION TITLE
Help finding already existing 3rd party extensions of JoomCCK

### DISCUSSION THREAD

**Discussion by andiamine** [JoomCoder-com/JoomCCK#89] [Ideas]
I need help here, if you know a 3rd party integration please share link here so we can notify developers of this extension that Cobalt CCK now is JoomCCK and they should update the integration, also this will help me to list all integrations with JoomCCK in documentation.

**Comment by Giorgi625** [comment]
I would like to have intergration with this extensions:
[Google Structured Data](https://www.tassos.gr/joomla-extensions/google-structured-data-markup)
[Perfect Publisher](https://www.extly.com/perfect-publisher.html) they had integration with CobaltCCK in their previous versions which were called Autotweet and Joocial.
Stackideas also had an integration with CobaltCCK
[OSMap](https://www.joomlashack.com/joomla-extensions/osmap/) also will be good to have sitemap integration.
and some opengraph integration if it will not be implemented into JoomCCK. IU remember you had created field for CobaltCCK for this feature long time ago which worked very well.
Jomsocial and Community Builder as well. 
and there are some others, if I remember I will write.

**Reply by andiamine** [reply]
Yes i still have those fields and packs, and will be added too soon :)

**Comment by andiamine** [comment]
Thanks @Giorgi625 
list will be listed here:
https://github.com/JoomCoder-com/JoomCCK/wiki/5.---Integrations

**Comment by Giorgi625** [comment]
I texted Extly to make integration with Perfect publisher. Also Tassos to add JoomCCK integration to Google Structured Data plugin.
Also texted to Joomlashack about integration with OSmap

**Reply by LordHelmchen666** [reply]
+1 for Osmap. Integration would be great. Here is latest Cobalt Xmap plugin. Maybe we could rewrite it?
[plg_cobalt.xmap.com_cobalt.j3.v.2.12.zip](https://github.com/JoomCoder-com/JoomCCK/files/14723748/plg_cobalt.xmap.com_cobalt.j3.v.2.12.zip)

**Reply by Giorgi625** [reply]
Got answer from Joomlashack: 

Hi, Giorgi,
Thank you for reaching out.
I'm sorry if this sounds disappointing, but we don't plan to integrate OSMap Pro with the JoomCCK.
Cordially,
Alex

**Reply by andiamine** [reply]
we are going to integrate it in Extended Version

**Reply by Giorgi625** [reply]
This is great news

**Comment by Giorgi625** [comment]
This is answer from Tassos:

Hello Giorgi,

Thank you for your suggestion.

We can certainly look into it as we get more requests.

In the meantime, you can use the [Menu Manager Integration](https://www.tassos.gr/joomla-extensions/google-structured-data/docs/the-menu-manager-integration) to add structured data to unsupported components.

Regards,
Tassos

**Reply by andiamine** [reply]
we will create richsnippets field you will don't need Google Structured Data

**Reply by Giorgi625** [reply]
Even better, will not need to install extra extension and spend money on it.

**Comment by Giorgi625** [comment]
Answer from Extly about integration with Perfect publisher:

Hi, Giorgi; thanks for reaching out.
Thanks for the heads-up. We'll add your request to our roadmap and work on this integration, depending on other customers' requests.
Regards,
